.github/workflows/lint.yml

```diff
@@ -56,7 +71,7 @@ jobs:
       submodules: true
       ref: ${{ github.event_name == 'pull_request' && github.event.pull_request.head.sha || github.sha }}
       script: |
-        export ADDITIONAL_LINTRUNNER_ARGS="--skip CLANGTIDY,CLANGFORMAT --all-files"
+        export ADDITIONAL_LINTRUNNER_ARGS="--skip CLANGTIDY,CLANGFORMAT,MYPY,MYPYSTRICT --all-files"
```

Comment:
Make sure that the mypy pip dependencies are up to date, I remember them being otu of date and may have implicitly listed in other jobs